src/layers.cpp

```diff
@@ -10,8 +10,9 @@ Key(*Layer_::getKey)(uint8_t layer, byte row, byte col) = Layer.getKeyFromPROGME
 
 // The total number of defined layers in the firmware sketch keymaps[]
 // array. If the keymap wasn't defined using CREATE_KEYMAP() in the
```

Comment:
KEYMAPS, not CREATE_KEYMAP

Reply:
See #253

Reply:
Oops. I was thinking of a different comment fix there. Sorry.